ingest/expression_files/dense_ingestor.py

```diff
@@ -128,14 +132,22 @@ def is_r_formatted_file(header, row):
             header (List[str]): Header of the dense matrix
             row (List): A single row from the dense matrix
         """
```

Comment:
this comment should be extended to mention the return value is a tuple, not just a boolean

Reply:
Although I don't see anywhere in the code that uses the second half of the tuple, should it be omitted?

Reply:
It's used in set_header()